src/components/PatchActionButtons/EnqueuePatch.tsx

```diff
@@ -1,78 +1,54 @@
-import React, { forwardRef } from "react";
-import { useMutation } from "@apollo/client";
+import React, { useState } from "react";
 import { Disclaimer } from "@leafygreen-ui/typography";
-import { Popconfirm } from "antd";
-import { usePatchAnalytics } from "analytics";
 import { DropdownItem } from "components/ButtonDropdown";
-import { useBannerDispatchContext } from "context/banners";
-import {
-  EnqueuePatchMutation,
-  EnqueuePatchMutationVariables,
-} from "gql/generated/types";
-import { ENQUEUE_PATCH } from "gql/mutations";
-import { StyledBody } from "./UnschedulePatchTasks";
+import { EnqueuePatchModal } from "pages/patch/index";
```

Comment:
You can just do `import { EnqueuePatchModal } from "pages/patch";`

Reply:
When I tried this it thought I was trying to import from `pages/Patch.tsx`